Ecosystem landing (#73)
scverse/scverse.github.io: content/_index.md

text = """\
**scverse** is a consortium of foundational tools (mostly in Python) for omics data in life sciences. It has been founded to ensure the long-term maintenance of these core tools."""

# Ecosystem

[ecosystem]

text = """\

> A broader ecosystem of packages builds on the **scverse** core packages. [These tools](/packages#ecosystem) implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, and more.

Reviewer: ```suggestion
A broader ecosystem of packages builds on the **scverse** core packages. [These tools](/packages#ecosystem) implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, and more."""
```

> Many packages rely on the functionality of the **scverse** core packages. They implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, etc. We are also building [a listing of such packages](/packages#ecosystem) that follow development best practices."""

Reviewer 2: ```suggestion
A broader ecosystem of packages builds on the **scverse** core packages. [These tools](/packages#ecosystem) implement models and analytical approaches to tackle challenges in spatial omics, regulatory genomics, trajectory inference, visualization, and more.
```
